src/main/java/net/openhft/chronicle/network/ConnectionStrategy.java

```diff
@@ -183,5 +183,12 @@ default void throwExceptionIfClosed() throws IllegalStateException {
             throw new ClosedIllegalStateException("Closed");
         }
     }
-    
+
+    default long minTimeSec() {
+        return 5;
```

Comment:
so I think these are the only methods that need to be added, and they should refer to the properties e.g. `Integer.getInteger("connectionStrategy.pause.min.secs", 5)`

I don't believe we need any changes to AlwaysStart... and FatalFailre...

Reply:
These should refer to the system properties. AlwaysStartOnPrimaryConnectionStrategy can refer to these thus ensuring system properties are in one place

Reply:
@foroughgoudarzi please see my last comment

Reply:
All system properties are in one place (AlwaysStartOnPrimaryConnectionStrategy). Please let me know where would you like to define these two system properties? ConnectionStrategy is an interface and its variable would be final and not configurable. I have added these system properties the same way that pauseMillisBeforeReconnect has been defined.

Reply:
We have to have the system property in the interface so that all implementations (even customer impls) use it, and we don't get behaviour changes
```
default long minTimeSec() {
        return Integer.getInteger("connectionStrategy.pause.min.secs", 5);
}
```
and in AlwaysStart... you can refer to the interface to default the field so we minimise duplication
```
private long minPauseSec = ConnectionStrategy.super.minTimeSec();
```
Don't forget to squash merge all these commits!